src/main/java/org/broadinstitute/hellbender/utils/OptimizationUtils.java

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.hellbender.utils;
+
+import org.apache.commons.math3.optim.MaxEval;
+import org.apache.commons.math3.optim.nonlinear.scalar.GoalType;
+import org.apache.commons.math3.optim.univariate.BrentOptimizer;
+import org.apache.commons.math3.optim.univariate.SearchInterval;
+import org.apache.commons.math3.optim.univariate.UnivariateObjectiveFunction;
+
+import java.util.function.Function;
+
+/**
+ * Created by davidben on 4/27/16.
+ */
+public class OptimizationUtils {
+    protected static final MaxEval DEFAULT_MAX_EVAL = new MaxEval(1000);
+    private static final double DEFAULT_RELATIVE_TOLERANCE = 0.001;
+    private static final double DEFAULT_ABSOLUTE_TOLERANCE = 0.001;
+    protected static final BrentOptimizer DEFAULT_OPTIMIZER = new BrentOptimizer(DEFAULT_RELATIVE_TOLERANCE, DEFAULT_ABSOLUTE_TOLERANCE);
+
+    protected static final MaxEval QUICK_MAX_EVAL = new MaxEval(100);
+    private static final double QUICK_RELATIVE_TOLERANCE = 0.01;
+    private static final double QUICK_ABSOLUTE_TOLERANCE = 0.01;
+    protected static final BrentOptimizer QUICK_OPTIMIZER = new BrentOptimizer(QUICK_RELATIVE_TOLERANCE, QUICK_ABSOLUTE_TOLERANCE);
+
+    private static final double EPSILON = 1e-5;
+
+    private OptimizationUtils() { }
+
+    public static double argmax(final Function<Double, Double> function, final double min, final double max, final double guess) {
+        final UnivariateObjectiveFunction objective = new UnivariateObjectiveFunction(x -> function.apply(x));
+        final SearchInterval interval= new SearchInterval(min, max, guess);
+        return DEFAULT_OPTIMIZER.optimize(objective, GoalType.MAXIMIZE, interval, DEFAULT_MAX_EVAL).getPoint();
+    }
+
+    public static double quickArgmax(final Function<Double, Double> function, final double min, final double max, final double guess) {
```

Comment:
I think it might be worth having a default method + a method with the ability to set max eval and the tolerances, rather than `argmax` + `quickArgmax`.